ava-test/cd.js

```diff
@@ -0,0 +1,103 @@
+import test from 'ava';
+import shell from '..';
+import path from 'path';
+import common from '../src/common';
+import fs from 'fs';
+import utils from './utils/utils';
```

Comment:
New format:

- core modules (alphabetized)
- npm modules (alphabetized)
- local files (alphabetized, but with `import shell from '..';` first because it needs to be loaded before the others)